x11-packages/emacs-pgtk/build.sh

```diff
@@ -0,0 +1,143 @@
+TERMUX_PKG_HOMEPAGE=https://www.gnu.org/software/emacs/
+TERMUX_PKG_DESCRIPTION="Extensible, customizable text editor-and more"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=29.0.60
+TERMUX_PKG_SRCURL=https://git.savannah.gnu.org/git/emacs.git
+# emacs-29 as of 2022-12-12
+TERMUX_PKG_GIT_BRANCH=emacs-29
+# TERMUX_PKG_GIT_BRANCH=19ef86f775a39eab46231c3eb6fc702e06857a5a
+# TERMUX_PKG_SHA256=75c2ea4dc648078bb88cb8b8980693d1c5a2d6cd077393dcab12b198c5b57e67
+TERMUX_PKG_DEPENDS="fontconfig, freetype, gdk-pixbuf, giflib, glib, harfbuzz, libgmp, libgnutls, libice, libjansson, libjpeg-turbo, libpng, librsvg, libsm, libtiff, libx11, libxaw, libxcb, libxext, libxfixes, libxft, libxinerama, libxml2, libxmu, libxpm, libxrandr, libxrender, libxt, littlecms, ncurses, zlib"
+TERMUX_PKG_CONFLICTS="emacs, emacs-x"
+TERMUX_PKG_REPLACES="emacs, emacs-x"
+TERMUX_PKG_PROVIDES="emacs-pgtk"
```

Comment:
The value of `TERMUX_PKG_PROVIDES` should be the same as others. Providing itself is no-op.

Reply:
This comment should have been addressed with the latest commits !